### DISCUSSION TITLE
Could you provide a configuration option, to not raise an exeption when helm uninstall happens, but the corresponding release is not installed?

### DISCUSSION THREAD

**Discussion by h0jeZvgoxFepBQ2C** [helmfile/helmfile#1802] [Ideas]
We are managing multiple customer releases and sometimes we add new releases to the config file, but dont deploy each customer.

Then sometimes we delete customers and helmfile runs the uninstall process, but it fails since it cannot find a corresponding release to uninstall - which should be skippable, and not raise an exception?

**Comment by h0jeZvgoxFepBQ2C** [comment]
Maybe also add a helmDefault option for uninstall process. When I set

```yaml
helmDefaults:
  args:
    - "--ignore-not-found=true"
```

it also doesnt work, since this option is passed to helm list, which doens't have this option flag and returns an error

**Comment by yxxhero** [comment]
@h0jeZvgoxFepBQ2C new feature.
